tools/distpartitioning/parmetis_preprocess.py

```diff
@@ -37,8 +37,14 @@ def get_proc_info():
         Rank of the current process.
     """
     env_variables = dict(os.environ)
-    return int(os.environ.get("PMI_RANK") or 0)
-
+    # mpich
+    if "PMI_RANK" in env_variables:
```

Comment:
when `PMI_RANK` is introduced, I ask a question for it. https://github.com/dmlc/dgl/pull/4756/files#r1009055022 could you take a look? so it turns out that `PMI_RANK` is not universally suitable?

Reply:
could you pls update the docstring above as it's kind of out-dated?

Reply:
Open MPI use OMPI_COMM_WORLD_RANK.
It seems there is no uniform NAME.

Reply:
all right.

Reply:
docstring updated.